Dockerfile

```diff
@@ -80,7 +80,7 @@ COPY requirements.txt /
 COPY tests/requirements.txt /test-requirements.txt
 # Since we used pip3 to install pip globally, pip should now be for Python 3
 RUN pip-compile --output-file /requirements.txt.lock /requirements.txt /test-requirements.txt
-RUN pip-sync /requirements.txt.lock
```

Comment:
Also, pip-sync ensures that the versions in the reqs are locked in, but that is hardly needed in a Dockerfile since a new image needs to be built anyway if there are changes to dependencies.